composables/useMyFetch.ts

```diff
@@ -0,0 +1,20 @@
+export const useMyFetch = async (request, opts) => {
+  const config = useRuntimeConfig()
+
+  try {
+    const {data,error} = await useFetch(request, { baseURL: config.public.baseURL, ...opts })
+    console.log("Fetch error:", data);
+    if (!data.value ) {
+      throw createError({
+        ...error.value,
+        statusMessage: `Could not fetch data.`,
+      });
+    }
+
+    return data;
+
+  } catch (error) {
+    console.error("Fetch error:", error);
```

Comment:
Just checking if we meant to leave this bit of code in or not? Thanks Ruby

Reply:
have removed console error

Reply:
Thank you